Source/Modules/main.cxx

```diff
@@ -37,6 +37,7 @@ int Verbose = 0;
 int AddExtern = 0;
 int NoExcept = 0;
 int SwigRuntime = 0;		// 0 = no option, 1 = -runtime, 2 = -noruntime
+int UseWrapperSuffix = 0;	// If 1, append suffix to non-overloaded functions too.
```

Comment:
is there an option to actually enable it ?

Reply:
Yes, in [C backend](https://github.com/swig/swig/pull/2086).

Reply:
oh, thanks